.gitignore

```diff
@@ -57,3 +57,4 @@ docs/_build/
 # PyBuilder
 target/
 python-rrmngmnt.iml
+/.venv/
```

Comment:
Please add comment that it is because of pycharm above this line.
I am not sure if it should not be just `.venv/` , without that first slash. But if it works I don't have a problem.

Reply:
I actually added the virtualenv dir to the .gitignore via Pycharm UI, and that's what it chose to add.
As you said - it's weird, but works :)